### DISCUSSION TITLE
Is it possible to configure the execution directory of task

### DISCUSSION THREAD

**Discussion by carlqt** [go-task/task#1689] [Q&A]
I'm wondering if it's possible to run the tasks in a Taskfile, outside of the current working directory, but the execution is relative to the current working directory?

If it's not clear, let's say I have this project layout. `foo` being the root directory of the project and `bar` is a directory that contains a `Taskfile.yml`.
```
.
└── foo/
    ├── docker-compose.yml
    ├── bar/
    │   └── Taskfile.yml
```

I've been trying to run the command using `-d` and `-t` flag e.g. `task -t bar/Taskfile.yml`.

I want to run a task from `bar/Taskfile.yml` but the execution directory remains in `foo`. I expect if the command is `pwd`, it displays the current directory where the command was executed.

Let me know if my question makes sense so I can improve it if not.

**Comment by andreynering** [comment]
Hi @carlqt,

Have you tried to run with both `-t` and `-d`? That should work.

You could also set `dir: '{{.ROOT_TASKFILE}}'` in that task.
